tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -302,6 +302,35 @@ def test_assign__dict(self, run):
 
         assert run["regular"].fetch() == run["stringified"].fetch()
 
+    def test_assign__drops_dict_entry_with_practically_empty_key(self, caplog, run):
```

Comment:
`stringify_unsupported` looks like doesn't have an impact here. Could we move this tests somewhere else?

Reply:
Done, moved to `test_handler.py`